code/game/objects/items.dm

```diff
@@ -587,7 +587,7 @@ var/global/image/fire_overlay = image("icon" = 'icons/effects/fire.dmi', "icon_s
 		var/turf/T = get_turf(src)
 		var/obj/effect/decal/cleanable/ash/A = new()

```

Comment:
the first arg of new is the loc.
so why isn't it ```T```?
that way DM itself will call Entered() and the entire call chain, properly.